SimplePagedViewFramework/Source/Views/PageDotsView.swift

```diff
@@ -148,6 +148,8 @@ private extension PageDotsView {
 
             if let previousDot = previousDot {
                 constraints.append(dot.leadingAnchor.constraint(equalTo: previousDot.trailingAnchor, constant: dotSize))
+            } else {
+                constraints.append(dot.leadingAnchor.constraint(equalTo: dotContainer.leadingAnchor))
```

Comment:
This is the only important file. The others are just me changing build settings so I could run the example on a real device.